docker/smqtk_iqr_playground/run_container.cpu.sh

```diff
@@ -11,7 +11,7 @@
 set -e
 
 # Container image to use
-IQR_CONTAINER=kitware/smqtk/iqr_playground
+IQR_CONTAINER=gitlab.kitware.com:4567/smqtk/smqtk-iqr/iqr_playground
```

Comment:
The GPU version of this could probably use a symmetric update. However, these shell scripts could also possibly be replaced with a docker-compose config to encapsulate their running, which would also then utilize the .env parameterizations instead of our replication/duplication of it here.

Reply:
Would this mean referencing the variables from the `.env` file here and in the GPU script? 

Like this `IQR_CONTAINER=${SMQTK_REGISTRY}/iqr_playground`?

Reply:
Not 100% what I was meaning, but I think that would be fine for the scope of this PR.